docs/participate/01_staking/01_become_a_collator/04_session_keys.md

```diff
@@ -6,11 +6,9 @@ title: Set and Rotate Session Keys
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Before a Collator can author blocks, the node needs to fully sync up with both the KILT parachain and the Kusama relaychain.
-Depending on the size of the blockchain states, it may take from a number of hours to few days for the node to fully synchronize.
-More details can be found on the [Polkadot network docs](https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-kusama#synchronize-chain-data).
-
-Once both chains have fully synced, the Collator needs to set sessions keys to begin collating on the network.
+The Collator needs to link session keys to his collator account.
+The session keys are identifying your node.
```

Comment:
Not sure if we need to repeat this concept in a different sentence. Maybe we can just merge it with the sentence above briefly.

```suggestion
With the keys linked, they are used to identify a collator's node.
```